### PR TITLE
Feature/easing between zoom levels

### PR DESCRIPTION
**Pull request by jedi-of-the-sea**
## Summary

With the new changes, the zoom between zoom levels works smoother now. 

## Instructions for local reproduction and review

Start a client and zoom in and out via zoom buttons or mouse.

I assumed that the smooth zoom is preferred and should be standard, so I just replaced the old zoom with the new method.

I'm not sure if the duration of the animation should be configurable, I just used a value of 1000. Let me know if this is something that I should add.

### CODE REVIEW

File: packages/plugins/Zoom/src/store/index.ts [hunk 1]
```diff
@@ -50,6 +51,15 @@ export const makeStoreModule = () => {
       decreaseZoomLevel({ dispatch, getters: { zoomLevel } }): void {
         dispatch('setZoomLevel', zoomLevel - 1)
       },
+      smoothZoom({ rootGetters: { map }, getters: { zoomLevel } }): void {
+        const view = map.getView()
+        if (view) {
+          view.animate({
+            zoom: zoomLevel,
+            duration: 1000,
```
Comment (dopenguin) [inline packages/plugins/Zoom/src/store/index.ts:59]:
```suggestion
            duration: 500,
```
A second felt a bit too slow

**Review by dopenguin — CHANGES_REQUESTED** [review]
🏓 @jedi-of-the-sea

**Comment by jedi-of-the-sea** [comment]
@dopenguin 🏓 I changed the duration to 500

**Review by dopenguin — APPROVED** [review]
🚀 @jedi-of-the-sea

**Pull request merged**